common/src/main/java/com/mx/common/models/ModelList.java

```diff
@@ -93,7 +93,7 @@ public final boolean remove(Object o) {
 
   @Override
   public final boolean containsAll(Collection<?> c) {
-    return container.containsAll(c);
+    return new HashSet<>(container).containsAll(c);
```

Comment:
This doesn't seem right.

Reply:
Oh yeah, My IntelliJ told me this would perform better. I wanted to see what you thought. I can revert that easily.

Reply:
Reverted